changelog.md

```diff
@@ -22,6 +22,14 @@ Prior versions of Marathon have tried to authenticate whenever a principal has b
 Framework authentication is now explicit. There is a command line toggle option for authentication: `--mesos_authentication`.
 This toggle is disabled by default. You must now supply this flag to use framework authentication.
 
+#### Changed default values for TASK_LOST GC timeout
+If a task is declared lost in Mesos, Marathon waits for the task to come back for a certain amount of time.
```

Comment:
Change to: `If a task is declared lost in Mesos, but the reason indicates it might come back, ...`